subprojects/docs/src/snippets/ant/useExternalAntTask/groovy/build.gradle

```diff
@@ -1,4 +1,4 @@
-task check {
+tasks.register('check') {
```

Comment:
That is true. However, the snippets was like this before `task check {}` would fail in a similar way. And the Kotlin DSL snippet is also like this.
This is rather old documentation about migrating Ant builds. We can probably improve this, but I think we then should look at the whole section and the text around this snippet.